source/templates.tex

```diff
@@ -7289,9 +7290,11 @@
 \begin{example}
 \begin{codeblock}
 template <class T, T> struct S {};
-template <class T> int f(S<T, T()>*);
-struct X {};
-int i0 = f<X>(0);
+template <class T> int f(S<T, T{}>*);   // \#1
+struct X {
```

Comment:
The resolution includes a change from `struct` to `class`, but the change was not marked up visibly.
```suggestion
class X {
```

Reply:
I thought that may have been related to https://github.com/cplusplus/draft/issues/5323, but [blaming](https://github.com/cplusplus/draft/blame/main/source/templates.tex#L7294) reveals that it's been a `struct` since it was first checked-in.

Reply:
It didn't look like this change (from struct to class) was intended to be part of the paper. Was it supposed to be? I'll leave this one for @tkoeppe.

Reply:
The change was absolutely intended to be part of the paper because the example would be wrong without that change.

Search for
> WIll replace struct with class and make a plain int x data member with an initializer.

in the CWG page for the Kona 2022 wiki.

Reply:
Ah ok, thanks. Fixed.